projects/samples/demos/worlds/hexapod.wbt

```diff
@@ -231,51 +230,51 @@ DEF HEXAPOD Robot {
         physics USE STANDARD_PHYSICS
       }
     }
-    DEF HIP_HINGE_JOINT_L0 HingeJoint {
+    DEF HIP_HINGE_JOINT_R0 HingeJoint {
       jointParameters HingeJointParameters {
-        axis 0 1 0
-        anchor 0.2 0 -0.05
+        axis 0 0 1
+        anchor 0.2 -0.05 0
       }
       device [
         RotationalMotor {
-          name "hip_motor_l0"
+          name "hip_motor_r0"
           maxVelocity 5
           minPosition -0.7
           maxPosition 0.7
         }
       ]
       endPoint Solid {
-        translation 0.2 0 -0.05
+        translation 0.2 -0.05 0
         children [
           USE HIP_SHAPE
           DEF LEFT_FEMUR_TRANS Transform {
```

Comment:
Shouldn't it be RIGHT_FEMUR_TRANS ?
Same remark for line 256.

If it is the case, the USEs have to be updated accordingly

Reply:
You are right.
I fixed some of the DEF names but not all of them